tests/nnapi/specs/V1_2/maximum_quant8.mod.py

```diff
@@ -0,0 +1,33 @@
+#
```

Comment:
Could you change this file name to `maximum_quant8_nnfw.mod.py`?
Our rule to create a new nnapi test file is to append `_nnfw`